gammacat/webpage.py

```diff
@@ -62,11 +62,26 @@ def make_references_data(self):
         references_data = []
         for reference_id in reference_ids:
             ads_url = f'https://ui.adsabs.harvard.edu/#abs/{reference_id}'
+            in_folder = f'https://github.com/gammapy/gamma-cat/blob/master/input/' \
```

Comment:
Can you please avoid the code duplication in these lines by putting the common things (e.g. the URL string, but also other lines look like copy & paste) into a temp local variable?